lib/utils/transcription-utils.js

```diff
@@ -845,7 +845,8 @@ module.exports = (logger) => {
         {hints: rOpts.hints}),
         ...(rOpts.hints?.length > 0 && typeof rOpts.hints[0] === 'object' &&
         {hints: JSON.stringify(rOpts.hints)}),
-        ...(typeof rOpts.hintsBoost === 'number' && {hintsBoost: rOpts.hintsBoost})
+        ...(typeof rOpts.hintsBoost === 'number' && {hintsBoost: rOpts.hintsBoost}),
+        callSid: task.cs.callSid
```

Comment:
This will break when custom stt is used as part of amd.  
https://github.com/jambonz/jambonz-feature-server/blob/d2a5d483d06491e4908af58a891a3a1cb8d1d6b2/lib/utils/amd-utils.js#L270

Reply:
> This will break when custom stt is used as part of amd.
> 
> https://github.com/jambonz/jambonz-feature-server/blob/d2a5d483d06491e4908af58a891a3a1cb8d1d6b2/lib/utils/amd-utils.js#L270

fixed by checking existence of task.cs.callSid